### ISSUE TITLE
{FEAT}: Requirement of GitHub Templates.

### ISSUE THREAD

**Issue by AbhijithGanesh** [piccolo-orm/piccolo#264]
We need requirement of templates to facilitate faster forms and issues.

**Comment by dantownsend** [comment]
Good idea. I don't much about GitHub issues templates - need to look into it.

**Comment by AbhijithGanesh** [comment]
If you need help, I have done this before many times and can get this done too.

**Comment by dantownsend** [comment]
I'm not sure what the template should contain. Do you have an example of another project which uses it well?

**Comment by AbhijithGanesh** [comment]
You check out 

 CiviWiki- The Templates were contributed by me

Eddiehub community - Another good example

**Comment by dantownsend** [comment]
OK, cool - if you want to open a PR with a bug template, that would be a nice start, and we can test it out.

**Comment by AbhijithGanesh** [comment]
I have opened the PR with three templates - bugs, issues and support form. You can review them and request changes
